JeffreyQ/UCLA-CS-130
# POST /user

This branch introduces a DB migration.
To test this branch:
* `make build` and `make run` to run the containers
* In a separate tab, run `make upgrade` to run the alembic upgrade script
* check out the docs at localhost:5000 to see how to make use of the POST /user endpoint
